### PR TITLE
Docs/add readme

### PR DESCRIPTION
**Pull request by MRowan121**
- Updated README file with contributor profile links, screenshot of application, and additional commentary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
-# idea-Box
\ No newline at end of file
+# idea-Box
+
+### `Abstract:`
+[//]: <> (Briefly describe what you built and its features. What problem is the app solving? How does this application solve that problem?)
```
Comment (RickV85) [inline README.md]: We can remove these prompts now.

File: README.md [hunk 2]
```diff
@@ -1 +1,61 @@
...
+Every developer has more ideas than time. As David Allen likes to say “the human brain is for creating ideas, not remembering them.” In this project, we’ll be building an application that records and archives our ideas (good and bad alike).
```
Comment (RickV85) [inline README.md:4]: Wow, I like the quote!

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
+
+Throughout the project, one of our focuses will be on providing a fluid and responsive client-side interface. To this end, we’ll rely on JavaScript to implement snappy filtering in the browser.
```
Comment (RickV85) [inline README.md:6]: Do we want to add anything about our localStorage feature so ideas are not lost on refresh? I feel like that is huge!

File: README.md [hunk 4]
```diff
@@ -1 +1,61 @@
...
+4. Craft code with clean style, using small functions that show trends toward DRYness and SRP.
+
+### `Technology Used:`
+[//]: <> (What tech did you work with?)
+The HTML, CSS, and JavaScript for this application was written in VS Code and pushed into GitHub. 
+
+### `Future Additions`
+1. The primary future addition planned for the Idea Box website would be for the user to be able to comment on their ideas.
+2. By favoriting an idea the user is able to in a sense already comment on their idea, but adding the ability to add additional text comments beyond whether they view the idea as a top idea or not would be beneficial.
+3. It would of course be important that this functionality persists beyond a single browser session just like the ideas themselves do
```
Comment (RickV85) [inline README.md]: I think we can remove this now that our localStorage is working.

**Review by RickV85 — CHANGES_REQUESTED** [review]
A few change requests on here. We need to be careful in the way we merge this branches with the refactoring branches that are open. We might just want to copy this doc to the latest branch once we all have reviewed and updated maybe?

**Review by RickV85 — CHANGES_REQUESTED** [review]
A few change requests on here. We need to be careful in the way we merge this branches with the refactoring branches that are open. We might just want to copy this doc to the latest branch once we all have reviewed and updated maybe?

File: README.md (hunk 2 above)
Comment (ElBrewster) [reply README.md:4]: Yea that's fancy, I like that.

**Review by ElBrewster — APPROVED** [review]
Looks good, great work guys.

**Pull request merged**